Tests/SonarScanner.MSBuild.Tasks.UnitTest/GetAnalyzerSettingsTests.cs

```diff
@@ -293,7 +305,7 @@ public void ConfigExists_ForProductProject_SonarAnalyzerSettingsUsed(string sona
 
             // Assert
             executedTask.RuleSetFilePath.Should().Be($@"c:\{language}-normal.ruleset");
-            executedTask.AnalyzerFilePaths.Should().BeEquivalentTo(@"c:\wintellect1.dll", @"c:\Google.Protobuf.dll", $@"c:\sonar.{language}.dll", @"c:\Google.Protobuf.dll");
+            executedTask.AnalyzerFilePaths.Should().BeEquivalentTo(@"c:\wintellect1.dll", @"c:\Google.Protobuf.dll", $@"c:\sonar.{language}.dll", @"c:\analyzer1.should.be.preserved.dll", @"c:\analyzer2.should.be.preserved.dll");
```

Comment:
review note: Google.Protobuf dll was duplicated

Reply:
It was as the previous behavior was to invoke the LegacyProject behavior so it picked up one version from one original analyzers list and the other from the other.

Reply:
can you update the UT name to reflect this tested behavior?

because `SonarAnalyzerSettingsUsed`  - by settings I understand AdditionalFiles and RuleSets, not analyzers...